portfolio/src/main/java/com/google/sps/servlets/fix.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Do you really want this file to be in this PR?

Reply:
oops! no indeed not 👍